src/io/flutter/sdk/FlutterSdkManager.java

```diff
@@ -110,23 +110,23 @@ default void flutterSdkRemoved() {
   // Listens for changes in Flutter Library configuration state in the Library table.
   private final class LibraryTableListener implements LibraryTable.Listener {
     @Override
-    public void afterLibraryAdded(Library newLibrary) {
+    public void afterLibraryAdded(@NotNull Library newLibrary) {
```

Comment:
I was curious, if the bots would catch it (I guessed they wouldn't)... These @NotNulls have been removed as well.  As soon as the Flutter plugin is on 2017.2.x, this cleanup can happen.